src/github/branch.ts

```diff
@@ -44,19 +44,25 @@ export const processBranch = async (
 
 	const jiraPayload: JiraBranchBulkSubmitData | undefined = await transformBranch(github, webhookPayload, logger);
 
+	const jiraClient = await getJiraClient(
```

Comment:
Can we move the jiraClient and conditional below the conditional to check if there's a jiraPayload (or move line 45 below the jiraClient stuffs?)

Reply:
Sure thing, I was thinking grouping the `buildX` code then grouping the exit early code.